### PR TITLE
Blimp: remove duplicated and unused params

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Comment by IamPete1** [comment]
need to also remove them from the .h

**Comment by Hwurzburg** [comment]
done...thanks Pete

File: Blimp/Parameters.h [hunk 1]
```diff
@@ -148,7 +148,6 @@ class Parameters
         // 140: Sensor parameters
         //
         k_param_compass,
-        k_param_frame_type,
```
Comment (IamPete1) [inline Blimp/Parameters.h:151]: these `k_param_` should be re-named `k_param_unused` or similar. Stops us reusing the index and loading the old param value.

Comment (Hwurzburg) [reply Blimp/Parameters.h:151]: I hope its correct now...first time playing with param removal

File: Blimp/Parameters.cpp [hunk 2]
```diff
@@ -102,7 +102,7 @@ const AP_Param::Info Blimp::var_info[] = {
     // @Param: FS_GCS_ENABLE
     // @DisplayName: Ground Station Failsafe Enable
     // @Description: Controls whether failsafe will be invoked (and what action to take) when connection with Ground station is lost for at least 5 seconds. See FS_OPTIONS param for additional actions, or for cases allowing Mission continuation, when GCS failsafe is enabled.
-    // @Values: 0:Disabled/NoAction,1:RTL,2:RTL or Continue with Mission in Auto Mode (Removed in 4.0+-see FS_OPTIONS),3:SmartRTL or RTL,4:SmartRTL or Land,5:Land (4.0+ Only)
+    // @Values: 0:Disabled/NoAction,5:Land (4.0+ Only)
```
Comment (MichelleRos) [inline Blimp/Parameters.cpp:105]: Perhaps we could remove the "4.0+ only" since Blimp doesn't have a pre-4.1 in any case?

Comment (Hwurzburg) [reply Blimp/Parameters.cpp:105]: no problem

**Review by MichelleRos — APPROVED** [review]
LGTM

File: Blimp/Parameters.cpp [hunk 3]
```diff
@@ -141,7 +141,7 @@ const AP_Param::Info Blimp::var_info[] = {
     // @Param: FLTMODE1
     // @DisplayName: Flight Mode 1
     // @Description: Flight mode when Channel 5 pwm is <= 1230
-    // @Values: 0:Stabilize,1:Acro,2:AltHold,3:Auto,4:Guided,5:Loiter,6:RTL,7:Circle,9:Land,11:Drift,13:Sport,14:Flip,15:AutoTune,16:PosHold,17:Brake,18:Throw,19:Avoid_ADSB,20:Guided_NoGPS,21:Smart_RTL,22:FlowHold,23:Follow,24:ZigZag,25:SystemID,26:Heli_Autorotate
+    // @Values: 0:LAND,1:MANUAL,2:VELOCITY,3:LOITER
```
Comment (rmackay9) [inline Blimp/Parameters.cpp:144]: I personally like lower case.. but not a blocker..

**Pull request merged**